examples/demo-rollup/src/txs_rpc.rs

```diff
@@ -137,14 +137,14 @@ mod tests {
 
         fn extract_relevant_txs(
             &self,
-            _block: Self::FilteredBlock,
+            _block: &Self::FilteredBlock,
         ) -> Vec<<Self::Spec as DaSpec>::BlobTransaction> {
             todo!()
         }
 
         fn extract_relevant_txs_with_proof(
```

Comment:
nit: This function isn't really useful anymore. I'd either remove it, or add a default implementation in the trait so that users don't have to do anything to implementing it.

Reply:
I added a default implementation